ext-php-rs-derive/src/function.rs

```diff
@@ -361,7 +362,7 @@ impl Function {
             })
             .collect::<Vec<_>>();
         let output = self.output.as_ref().map(|(ty, nullable)| {
-            let ty: Type = syn::parse_str(ty).unwrap();
+            let ty: Type = syn::parse_str(ty).expect("failed to parse ty: function.rs:365");
```

Comment:
Isn't the filename/line number be part of the panic?

Reply:
Yep 😆